server/src/main/java/io/druid/segment/realtime/firehose/IngestSegmentFirehose.java

```diff
@@ -77,7 +76,7 @@ public IngestSegmentFirehose(
                             Filters.toFilter(dimFilter),
```

Comment:
How else would this be turned into a `Sequence<InputRow>` rather than `Sequence<Sequence<InputRow>>`?